src/Datadog.Trace/PlatformHelpers/AzureAppservices.cs

```diff
@@ -75,59 +75,78 @@ static AzureAppServices()
 
         public AzureAppServices(IDictionary environmentVariables)
```

Comment:
I just realized the casing of the filename isn't correctly cased. I wonder if this is easy to change with Git but if it's hard then just disregard

Reply:
I've noticed this for a while actually and it is bothersome.
It's annoying to change case directly, but I had an idea to do 2 commits to force git the rename and respect the case.
Done.